doc: post 1.9.0 doc update

b-gent opened the pull request:
Post-release updates to docs.

Signed-off-by: Bartosz Gentkowski <bartosz.gentkowski@nordicsemi.no>

b-gent commented:
Things to do as soon as #6707  and #6812 are merged:

- add known issues back to index
- remove orphan from known issues
- add 1.9.99 to versions.json
- update SES links in links.txt
- add changelog to release notes index

greg-fer commented:
> add changelog to release notes index

Also, @b-gent , check that the changelog is cleaned up and/or contains only entries for 1.9.99.

b-gent commented:
> 

Changelog is being cleaned by Uma in the release notes PR.

umapraseeda commented on doc/nrf/release_notes.rst:
```diff
@@ -12,13 +12,15 @@ This page is included only in the latest documentation, because it might contain
    A "99" at the end of the version number of this documentation indicates continuous updates on the main branch since the previous major.minor release.
    When looking at this latest documentation, be aware of the following aspect:
```
```suggestion
   When looking at this latest documentation, be aware of the following aspects:
```

b-gent replied:
done

umapraseeda commented on doc/nrf/release_notes.rst:
```diff
@@ -12,13 +12,15 @@ This page is included only in the latest documentation, because it might contain
    :maxdepth: 1
    :caption: Subpages:
 
+   releases/release-notes-changelog
```
Also, we need to remove the :orphan: tag from release-notes-changelog

b-gent replied:
Done now, thought I did this but must have not committed

greg-fer reviewed (approved):
Changelog was cleaned in the release notes PR, so this looks good to go for me.

umapraseeda commented on doc/nrf/releases/release-notes-changelog.rst:
```diff
@@ -131,21 +129,21 @@ Zephyr
+   git log --oneline manifest-rev ^45ef0d2
 
-The current |NCS| main branch is based on the Zephyr v2.7 development branch.
+The current nRF Connect SDK main branch is based on revision ``45ef0d2`` of Zephyr, which is located between v2.7.0 and v3.0.0 of Zephyr.
```
```suggestion
The current |NCS| main branch is based on revision ``45ef0d2`` of Zephyr, which is located between v2.7.0 and v3.0.0 of Zephyr.
```

umapraseeda reviewed (approved):
Just a minor fix..LGTM otherwise.

umapraseeda commented:
Also, do we need the mcumgr section in the 1.9.99 changelog as we had removed it  in the release notes for 1.9, @carlescufi or @shantha-14 ?

carlescufi commented:
> Also, do we need the mcumgr section in the 1.9.99 changelog as we had removed it in the release notes for 1.9, @carlescufi or @shantha-14 ?

No, mcumgr is no longer a separate project but rather part of Zephyr, so this section is no longer required.

umapraseeda commented:
> > Also, do we need the mcumgr section in the 1.9.99 changelog as we had removed it in the release notes for 1.9, @carlescufi or @shantha-14 ?
> 
> No, mcumgr is no longer a separate project but rather part of Zephyr, so this section is no longer required.

Could you remove that section @b-gent ?

b-gent commented:
> > > Also, do we need the mcumgr section in the 1.9.99 changelog as we had removed it in the release notes for 1.9, @carlescufi or @shantha-14 ?
> > 
> > 
> > No, mcumgr is no longer a separate project but rather part of Zephyr, so this section is no longer required.
> 
> Could you remove that section @b-gent ?

removed

grochu commented:
Hi @umapraseeda, I posted this PR: https://github.com/nrfconnect/sdk-nrf/pull/6935 to fix release notes highlights - I overlooked an issue during the Release Notes review. If you prefer, the change can be incorporated to this PR. I think it is important to fix this statement to avoid misleading information.
